infrastructures/kubernetes/src/test/resources/jwtproxy-confg.yaml

```diff
@@ -3,38 +3,43 @@ jwtproxy:
   - listen_addr: :8080
     verifier:
       upstream: http://tomcat:8080/
-      audience: http://workspace123
+      audience: workspace123
       max_skew: 1m
-      max_ttl: 3h
+      max_ttl: 8800h
       key_server:
         type: preshared
         options:
           issuer: wsmaster
-          key_id: mykey
+          key_id: workspace123
           public_key_path: /config/mykey.pub
       claims_verifiers:
       - type: static
         options:
           iss: wsmaster
       nonce_storage:
         type: void
+      excludes:
```

Comment:
I would say that it is better to remove it here, to check a use case when config doesn't contain any excludes. I mean testing of the following if statement https://github.com/eclipse/che/pull/10463/files#diff-8e4d2e1f45436acbfd7d188ac3f947beR71

Reply:
Ok